packages/did/src/DidBatcher/FullDidCreationBuilder.ts

```diff
@@ -127,9 +133,17 @@ export class FullDidCreationBuilder extends FullDidBuilder {
 
     this.consumed = true
 
+    const { publicKey, type } = this.authenticationKey
+
+    const pk =
+      type === VerificationKeyType.Ecdsa && publicKey.length > 32
```

Comment:
C: since you are using it twice in this file and similarly also in the DemoKeyStore, could this maybe be a little util function?

Reply:
https://github.com/KILTprotocol/sdk-js/pull/478/commits/8b62e3aad995ee0c1332052a536a18e3d037a0ce

The problem of using one function for both cases is that in the builder we deal with `DidVerificationKeyType`s, while in the demo keystore we deal with `SigningAlgorithms`, which can potentially be unrelated to DIDs. So we must declare two separate functions for both cases. I have declared a utility function for the DID case in this commit, so I guess all is good now 👍